test_scenarios/T9_Submit_A_CN.md

```diff
@@ -0,0 +1,79 @@
+# Test Scenario
+
+## T9 Submit a new CN
+
+### Warunki początkowe 
+
+### Przypadki testowe
+
+#### 1. Poprawne zarejestrowanie maszyny
+
+___WARUNKI POCZĄTKOWE:___
+
+- __Supplier__ skonfigurował maszynę do rejestracji.
+- __Supplier__ jest zalogowany jako `userT9_1`.
+- __Supplier__ znajduje się w detalach maszyny `mashineT9_1`. 
+
+1. __Supplier__ uruchamia na maszynie rejestrację z poprawnie
+ustawionym tokenem i parametrami od `mashineT9_1`.
+2. System wyświetla po pewnym czasie informację "Successfully submited".
```

Comment:
Skad taki komunikat miałby się wziąć? Według mnie po uruchomieniu naszego klienta na maszynie powinniśmy tylko zobaczyć zmieniony status maszyny.

Reply:
Tak, lepszą frazą byłoby "wypisze"...

Reply:
No to git, zgadza sei